### DISCUSSION TITLE
FreeCAD is Crazy! 0% Intuitive.

### DISCUSSION THREAD

**Discussion by juanpc2018** [FreeCAD/FreeCAD#11324] [Discussions that will turn in to issues but need refining]
im not an idiot, if i need to see 10 tutorial videos,
software is wrong...

№ 1 it should have a guided automated tutorial,
like a movie inside FreeCAD,
moves the mouse does something, pop-up window appears to explain,
and has a continue button, 
some times the pop-up window will ask user to do things,
until the 3D model is complete...
different 3D models for increased difficulty, and increased duration.
easy 3D tutorial, 5minutes, 
2nd 10 minutes,
3rd 15 minutes,
etc....

№ 2 
CNC software is light years ahead...
download [MillWizard](https://www.machsupport.com/software/mill-wizard-beta/), to see how easy is to do basic shapes.

[KiCAD](https://www.youtube.com/watch?v=D3it8wyJef0&t=1s) also better, for PCB Design.

Parametric Design = NLE  NDE, Non Linear Editing, Non Destructive Editing.
Grid should be like Audio software, ProTools, [Audacity](https://www.youtube.com/watch?v=xgdYuSHdkso&t=20s), KiCAD, GIMP, InkScape, Davinci Resolve Video Editor.

Quad Window View, X,Y,Z, 45°
[Insert Image](https://www.youtube.com/watch?v=EMnUxjyJ4Fw) - [Add Reference Picture](https://www.youtube.com/watch?v=j6i_W8CDgco&t=1008s)

https://www.youtube.com/watch?v=5AKE8P0YneE&t=41s

**Comment by luzpaz** [comment]
Thanks for your feedback. We are all volunteers here taking on an enormous task of building a CAD/CAM/FEA tool. If you have any way to support the effort, please feel free. If you want to push the effort further please participate on our [bugtracker](https://github.com/FreeCAD/FreeCAD/issues) and help test fixes and UI changes. If you know how to create tutorial clips and such please let us know. If you have a CnC machine that you'd be willing to test on, let us know. If you can donate a Path dev a CnC machine, let us know.

We're currently working on improving toponaming, building an assembly workbench, improving the UI/UX, improving Sketcher for the upcoming version 1.0 release. 

Note: i edited your OP because you used notation that was linking to closed tickets in the bugtracker.

**Comment by luzpaz** [comment]
I'm going to close this discussion because it's very broad.
